moto/events/responses.py

```diff
@@ -62,7 +62,7 @@ def describe_rule(self):
         rule = self.events_backend.describe_rule(name)
 
         if not rule:
-            return self.error("ResourceNotFoundException", "Rule test does not exist.")
+            return self.error("ResourceNotFoundException", "Rule " + name + " does not exist.")
```

Comment:
`return self.error("ResourceNotFoundException", "Rule {} does not exist.".format(name))`
is more pythonic.

Reply:
I agree, but I felt it was better to keep consistency with the rest of the file.